uaclient/entitlements/tests/test_fips.py

```diff
@@ -88,7 +87,7 @@ def test_enable_configures_apt_sources_and_auth_files(self, entitlement):
                     entitlement.name
                 ),
                 repo_url,
-                "TOKEN",
+                "%s-token" % entitlement.name,
```

Comment:
A nit, but `.format` here would be more consistent with the rest of the codebase.